server/src/controllers/questionController.js

```diff
@@ -16,7 +16,7 @@ class QuestionController {
       createdBy: req.body.user,
       meetup: req.body.meetup,
       title: req.body.title,
-      body: req.body['body of question'],
+      body: req.body.body,
```

Comment:
why use body?. Don't you feel question would have been better?. 'req.body.body' can be very confusing

Reply:
Okay. Thank you for your contribution. I'll implement that